### DISCUSSION TITLE
Prompt a List of a List<int>

### DISCUSSION THREAD

**Discussion by DrHeinzDoofenshmirtz** [shibayan/Sharprompt#201] [Q&A] [answered]
Hi.

I have a question regarding prompting a list of a list.
Is it possible to something like this?
`Prompt.List<List<int>>("What is your favorite integer numbers?")`

It doesn't work at the moment, as I get the following error message
`CollectionConverter cannot convert from System.String.`

I have tried inputting
`1,2,3`, `[1,2,3]` and `{1,2,3}`
but all of them returns the same error message.

Is there some way that this can be possible? A List of Enums could be replaced by something like the MultiSelect, just for ideas.

I hope someone can help.

Best, Matt

**Comment by shibayan** [comment, marked as answer]
That would be a usage that is not currently supported. .NET type conversion logic does not support string to array conversion.

It may be possible to create a custom converter as shown in the SO answer below.
https://stackoverflow.com/questions/34330017/convert-string-to-array-of-string-using-typeconverter

**Reply by DrHeinzDoofenshmirtz** [reply]
I actually find a really good solution.
I just made a loop and just prompted a list inside it multiple times. The result was then put into a List<List<int>>. :)
